omnigibson/prims/material_prim.py

```diff
@@ -28,6 +28,32 @@ class MaterialPrim(BasePrim):
             mtl_name (None or str): If specified, should be the name of the mtl preset to load.
                 None results in default, "OmniPBR"
     """
+    # Persistent dictionary of materials, mapped from prim_path to MaterialPrim
+    MATERIALS = dict()
```

Comment:
I think at some point we still need to get a global prim registry instead of doing this ad-hoc this way for a particular prim type. Should we do that now? Or maybe you can open an issue.

Reply:
Makes sense. Let's do this in the next PR. I opened an issue.